study-builder/studies/brain/release.conf

```diff
@@ -27,7 +27,9 @@
   "translatedSubtitles": [
```

Comment:
RELEASE is  existing activity. We cannot deploy the activity again. It has to be  a patch to update existing activity template text

Reply:
Added patch code to look at updated release.conf and update study config with styling changes